### PR TITLE
Updating the course with url link to pluralsight

### PR DESCRIPTION
**Pull request by adhithiravi**
Updating my Pluralsight course Cypress End to End testing link.

### CODE REVIEW

File: source/_data/courses.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@
   sourceUrl: https://www.udemy.com/course/cypress-test-automation-for-people-in-a-hurry/?referralCode=21A875FF429457CF641D
   
 - title: "Cypress: End-to-end JavaScript Testing"
+  url: https://app.pluralsight.com/library/courses/cypress-end-to-end-javascript-testing/table-of-contents
```
Comment (bahmutov) [inline source/_data/courses.yml:11]: ohhh, I am so sorry for leaving the most important part out 🙀

**Comment by bahmutov** [comment]
@adhithiravi could you sign the CLA and I will merge it

**Comment by adhithiravi** [comment]
Done.

**Comment by adhithiravi** [comment]
I don't think I have access to merge.

**Pull request merged**